app/views/candidate_mailer/_gain_insights_into_life_as_a_teacher.text.erb

```diff
@@ -3,3 +3,5 @@
 Respond to realistic video scenarios to understand what life in the classroom is like. Get real-time feedback from experienced teachers on your strengths as a potential teacher.
 
 [Try the realistic job preview tool](<%= candidate_realistic_job_preview_link(@application_form.candidate) %>)
+
+[Find out more about life as a teacher on the Get Into Teaching website](<%= t('get_into_teaching.url_life_as_a_teacher') %>). Explore the rewards of teaching, including pay, career growth, support, and the range of subjects and pupils you could teach.
```

Comment:
The GIT links don't seem to work when I click on them, I have to click and open in a new tab for it to work. Is this just a review app thing, will it work in production?

Reply:
Is this something we are planning to fix?